source/test/TestApp/TestUtils.cpp

```diff
@@ -3,8 +3,44 @@
 #include "TestAppPch.h"
 #include "TestUtils.h"
 
+#include <random>
+#include <sstream>
+
 namespace PlayFabUnit
 {
+    
+    std::string GenerateUuidV4() {
+        std::random_device              rd;
+        std::mt19937                    gen(rd());
```

Comment:
std::mt19937 defines a "source" with 32 bits of randomness.
https://en.cppreference.com/w/cpp/numeric/random/mersenne_twister_engine

Reply:
The simplest improvement is to convert to the 64 bit mt19937_64 option.
Upgrading that to 128 bits is awkward though, as it requires creating two of these, in a way where one isn't biased against the other. I think that it requires putting a sleep between the two, such as a 1ms sleep (which will be randomized by how long the operating system actually chooses to sleep, likely >1ms). That MAY be "sufficient" to get "more than" 64 bits of randomness, but I'm not sure it'll be the full 128 bits. You might have to do some research on how the seed is generated in order to get randomness at the full promised 128 bits.

Reply:
I pulled this info from stack overflow. I'm leaving a note about the source and moving on. This is sufficient for the test we are adding.